addons/website_livechat/static/src/bugfix/public_bugfix.scss

```diff
@@ -0,0 +1,22 @@
+/**
+* This file allows introducing new styles without contaminating other files.
+* This is useful when bug fixing requires adding new components for instance in
+* stable versions of Odoo. Any style that is defined in this file should be isolated
+* in its own file in master.
+*/
```

Comment:
Hello, just a quick comment, please don't add back this file, they all have been removed in v15 as files are now defined in manifest and not in a view (that previously would required a migration script to add).